Tasks/dratsevich/task_01/task_01.py

```diff
@@ -0,0 +1,10 @@
+amount = float(input('input amount of money \n'))
+time = int(input('input time in years \n'))
+rate_deposit = float(input('input rate deposit \n'))
+capitalisation = input('do you want to enable monthly capitalisation? (yes/no)')
```

Comment:
it's a money-related task, so I advise to use the Decimal data type for the calculation, -1 point

Reply:
but you said that round() works like "bank system round" so i used round. and it works correctly :)